XamlControlsGallery/ControlPages/CommandBarPage.xaml

```diff
@@ -44,21 +44,22 @@
     </Page.Resources>
     <StackPanel>
         <local:ControlExample x:Name="Example3" HeaderText="A command bar with labels on the side free floating in a page" >
+            <StackPanel>
+                <TextBlock x:Name="SelectedOptionText"></TextBlock>
```

Comment:
I agree with @chingucoding. Other samples (like CommandBarFlyout) display the selected option below the content. I think we should do the same here - please move this to below the CommandBar and consider adding more padding.

Reply:
Awesome, will do now